kernel-crawler/Dockerfile

```diff
@@ -5,9 +5,9 @@ COPY main.go /go/src/main.go
 RUN CGO_ENABLED=0 go build -o /go/bin/rhel-crawler /go/src/main.go
 
 ARG p7zip="v17.03"
-RUN wget "https://github.com/p7zip-project/p7zip/archive/refs/tags/${p7zip}.tar.gz" \
- && mkdir p7zip && tar xzf "${p7zip}.tar.gz" -C p7zip --strip-components=1 \
- && cd p7zip && make -j 4 all3 && make install
+RUN apt-get update && apt-get install unzip && \
+ wget "https://github.com/p7zip-project/p7zip/releases/download/${p7zip}/linux-cmake-p7zip.zip" \
+ && unzip linux-cmake-p7zip.zip -d /p7zip
```

Comment:
I'd like to understand why we need the `17.03` version of pzip and why we can't directly use the one installed with `apt-get`.  @robbycochran do you have any context on this?